Domains/Smartmetering/smartmeteringwebservices/GetGetMBusEncryptionKeyStatusResponse.md

```diff
@@ -0,0 +1,12 @@
+## GetGetMBusEncryptionKeyStatusResponse request
+
+### Description
+GetGetMBusEncryptionKeyStatusResponse is a request to return the M-Bus encryption key status as requested by a [GetMBusEncryptionKeyStatus](GetMBusEncryptionKeyStatus.md) request. 
```

Comment:
I think a reference to EncryptionKeyStatus in sm-configuration.xsd would be helpfull